sql/src/main/java/io/crate/expression/scalar/SubscriptRecordFunction.java

```diff
@@ -21,60 +21,44 @@
 
 package io.crate.expression.scalar;
 
-import java.util.List;
-
 import io.crate.data.Input;
 import io.crate.data.Row;
-import io.crate.expression.symbol.FuncArg;
 import io.crate.metadata.FunctionIdent;
-import io.crate.metadata.FunctionImplementation;
 import io.crate.metadata.FunctionInfo;
-import io.crate.metadata.FunctionResolver;
 import io.crate.metadata.Scalar;
 import io.crate.metadata.TransactionContext;
-import io.crate.types.DataType;
+import io.crate.metadata.functions.Signature;
 import io.crate.types.DataTypes;
 import io.crate.types.RowType;
 
+import javax.annotation.Nullable;
+import java.util.List;
+
 public final class SubscriptRecordFunction extends Scalar<Object, Object> {
 
     public static final String NAME = "_subscript_record";
+    public static final Signature SIGNATURE = Signature.scalar(
+        NAME,
+        RowType.EMPTY.getTypeSignature(),
```

Comment:
We do support the type singature parsing for `record(e v [, e v, ...] )`. It seems that we also do not need it for this case, see the previous behaviour https://github.com/crate/crate/pull/9919/files#diff-043c3fea4ce68fa7312a4150d7fb1ab4L54. I am not sure whether it is fine or we must tackle this support now.